jwst/srctype/srctype.py

```diff
@@ -156,6 +156,14 @@ def set_source_type(input_model):
         log.info(f'Input is a TSO exposure; setting default SRCTYPE = {src_type}')
         input_model.meta.target.source_type = src_type
 
+    # FOR NIR_WFSS check slit values of is_star to set SRCTYPE
```

Comment:
This needs to be done for all WFSS modes, so that includes `NRC_WFSS` and `NIS_WFSS`. This means that some of the change log entries above need to be updated to indicate that these changes apply to WFSS in general, not just NIS_WFSS.

Reply:
I am not all the familiar with WFSS mode. I did a search on WFSS in the pipeline and frequently get these modes:
WFSS_EXPTYPES = ['NIS_WFSS', 'NRC_WFSS', 'NRC_GRISM', 'NRC_TSGRISM']
NRC_TSGRISM is a mode that is already handled in the srctype code. But I could not find NRC_GRISM 
Do I need to add it to this   too ?